test/FlatLaunchpeg.test.ts

```diff
@@ -63,28 +63,36 @@ describe('FlatLaunchpeg', () => {
   beforeEach(async () => {
     config = { ...(await getDefaultLaunchpegConfig()) }
     await deployFlatLaunchpeg()
-    await flatLaunchpeg.setPublicSaleActive(true)
   })
 
   describe('Initialization', () => {
+    it('Phases can be initialized only once', async () => {
+      await initializePhasesFlatLaunchpeg(flatLaunchpeg, config, Phase.NotStarted)
+      await expect(initializePhasesFlatLaunchpeg(flatLaunchpeg, config, Phase.DutchAuction)).to.be.revertedWith(
```

Comment:
We should try not to use dutch auction properties in the FlatLaunchpeg tests, to avoid confusion. I think you can just use Phase.NotStarted here?